src/formatDate.js

```diff
@@ -8,7 +8,89 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let result = '';
+
+  let symbol = '';
+
+  for (let i = 0; i < date.length; i++) {
+    if (isNaN(date[i])) {
+      symbol = date[i];
+    }
+  }
```

Comment:
The loop to find the separator can be avoided by using the `fromFormat[3]` directly since it contains the separator.